<?xml version="1.0" encoding="UTF-8"?>
<!--

***********************************************************************************
* Copyright (C) 2019 - 2021, BlockSettle AB
* Distributed under the GNU Affero General Public License (AGPL v3)
* See LICENSE or http://www.gnu.org/licenses/agpl.html
*
**********************************************************************************

-->
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BlockSettle Settings</string>
  </property>
  <property name="sideNav" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Fixed</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::SinglePass</enum>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="batchSize">
         <number>100</number>
        </property>
        <property name="currentRow">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>GENERAL</string>
         </property>
         <property name="toolTip">
          <string>General application settings</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>46</red>
            <green>51</green>
            <blue>53</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="icon">
          <iconset resource="../armory.qrc">
           <normaloff>:/resources/general-settings@1.5x.png</normaloff>
           <selectedoff>:/resources/general-settings-selected@1.5x.png</selectedoff>:/resources/general-settings@1.5x.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NETWORK</string>
         </property>
         <property name="toolTip">
          <string>Network connection settings</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="0">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="icon">
          <iconset resource="../armory.qrc">
           <normaloff>:/resources/network@1.5x.png</normaloff>
           <selectedoff>:/resources/network-selected@1.5x.png</selectedoff>:/resources/network@1.5x.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SIGNER</string>
         </property>
         <property name="toolTip">
          <string>Signer settings</string>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="0">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="icon">
          <iconset resource="../armory.qrc">
           <normaloff>:/resources/password-icon@1.5x.png</normaloff>:/resources/password-icon@1.5x.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DEALING</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="0">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="icon">
          <iconset resource="../armory.qrc">
           <normaloff>:/resources/dealing@1.5x.png</normaloff>
           <selectedoff>:/resources/dealing-selected@1.5x.png</selectedoff>:/resources/dealing@1.5x.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>API</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>243</red>
            <green>244</green>
            <blue>245</blue>
           </color>
          </brush>
         </property>
         <property name="icon">
          <iconset resource="../armory.qrc">
           <normaloff>:/resources/api@1.5.png</normaloff>
           <selectedoff>:/resources/api-selected@1.5.png</selectedoff>:/resources/api@1.5.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="GeneralSettingsPage" name="pageGeneral"/>
        <widget class="NetworkSettingsPage" name="pageNetwork"/>
        <widget class="SignerSettingsPage" name="pageSigner"/>
        <widget class="DealingSettingsPage" name="pageDealing"/>
        <widget class="APISettingsPage" name="pageAPI"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="buttonBar" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSetDefault">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Restore Default</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeneralSettingsPage</class>
   <extends>QWidget</extends>
   <header>GeneralSettingsPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NetworkSettingsPage</class>
   <extends>QWidget</extends>
   <header>NetworkSettingsPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SignerSettingsPage</class>
   <extends>QWidget</extends>
   <header>SignerSettingsPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DealingSettingsPage</class>
   <extends>QWidget</extends>
   <header>DealingSettingsPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>APISettingsPage</class>
   <extends>QWidget</extends>
   <header>APISettingsPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../armory.qrc"/>
  <include location="../armory.qrc"/>
 </resources>
 <connections/>
</ui>
